.github/workflows/notion-pr-sync.yml

```diff
@@ -0,0 +1,55 @@
+name: Notion PR Sync
```

Comment:
Is this action still required?

Reply:
Still trying to get confirmation from the engineering leads channel, will chase again

Reply:
@sivashanmukh, as per our discussion on a thread almost a month ago, we don't need this anymore, right?

Reply:
Resolving for now to unblock the PR.